test/adagrams_test.rb

```diff
@@ -143,7 +147,8 @@
       # verify both have a score of 10
       expect(score_word(words.first)).must_equal 18
       expect(score_word(words.last)).must_equal 18
-
+      require "pry"
+      binding.pry
```

Comment:
please please please! Take all the `binding.pry`'s out of your code before submission!